cloudahoywriter.lua

```diff
@@ -200,6 +201,17 @@ local function simTime_to_recordingTime(simTime)
     return simTime - recordingStartSimTime
 end
 
+-- Converts dots (in the range [-2.5, 2.5]) to a scale of [-1, 1].
+local function dots_to_ones(dots)
```

Comment:
math.clamp?

Reply:
> math.clamp?

Not available in the math lib for Lua 5.1